README.md

```diff
@@ -1,2 +1,54 @@
 # action-baseline
-A GitHub Action for running the OWASP ZAP Baseline scan
+A GitHub Action for running the OWASP ZAP Baseline scan. 
+
+The ZAP baseline scan action scans a target URL for vulnerabilities and create issue in the relevant Github Repository
+with the link to the latest report. The generate reports will be committed to repository.
+
+Checkout the `action.yml` file for additional parameters on how to use it.
+
+Optionally you can specify a config file to modify the behaviour of ZAP scan to ignore certain warnings.
+
+## Usage
+
+** Basic **
+```
+steps:
+  - name: ZAP Scan
+    uses: zap/action-baseline@v1
+    with:
+      token: ${{ secrets.GIT_TOKEN }}
+      target: 'https://www.zaproxy.org/'
+```
+
+** Advanced **
+
+```
+on: [push]
+
+jobs:
+  zap_scan:
+    runs-on: ubuntu-latest
+    name: Scan the webapplication
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: master
+      - name: ZAP Scan
+        uses: zap/action-baseline@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          docker_name: 'owasp/zap2docker-stable'
+          target: 'https://www.example.com'
+          rules_file_name: '.zap/rules.tsv'
+```
+
+## Publish a new version
+
+```bash
+npm run package
+git add action.yml dist/index.js node_modules/*
```

Comment:
This needs tweak, we can also use `ncc` to include the dependencies.

Reply:
Okay, I'll check, but I could not find a param in the lib. 
The official blog also suggests the [same approach](https://help.github.com/en/actions/building-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github)

BTW I guess we need to remove this content as it is displayed in the market place as well.
wdyt @thc202 ?

Reply:
I was suggesting following:
> As an alternative to checking in your node_modules directory you can use a tool called zeit/ncc to compile your code and modules into one file used for distribution.

We do have to push `dist` though (which seems to be the recommended, based on official actions e.g. https://github.com/actions/checkout ).

Agreed, we should move this to other file.

Reply:
@thc202 , yes the PR also uses the `zeit/ncc` . Thus it only adds the node modules to the release instead of the master branch.

![image](https://user-images.githubusercontent.com/13045528/78533691-a35ba900-77e9-11ea-922b-fbfacdadd7ac.png)